MAUI/Masked-Entry/Overview.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: About .NET MAUI MaskedEntry control | Syncfusion
+description: Learn all about the introduction of Syncfusion MAUI MaskedEntry (SfMaskedEntry) control with essential features and more here.
+platform: maui
+control: SfMaskedEntry
+documentation: ug
+---
+
+# .NET MAUI MaskedEntry Overview
+
+The `.NET MAUI MaskedEntry` is an advanced version of the input control that restricts input of certain characters, text, and numbers by using a mask pattern. This control is used to create a template for providing information such as telephone numbers, email IDs, IP addresses, product keys, and so on.
+
+![MaskedEntry control overview in MAUI](MaskedEntry_Images/maui_maskedentry_overview.png)
+
+## Key features
+
+* `MaskTypes`: The input can be masked with a fixed or variable length by setting the mask type to simple or regex, respectively.
```

Comment:
* `MaskTypes`: The input can be masked with a fixed or variable length by setting the mask type to simple or regex.